docs/using-cinder-csi-plugin.md

```diff
@@ -434,6 +434,24 @@ Following prerequisites needed for volume cloning to work :
 
 Sample yamls can be found [here](https://github.com/kubernetes/cloud-provider-openstack/tree/master/examples/cinder-csi-plugin/clone)
 
+### Multi-Attach Volumes
+
+To avail the multiattach feature of cinder, specify the 'type' of the volume as 'multiattach' in storageclass definition.
+This should enable to attach a volume to multiple hosts/servers simultaneously.
+
+Example:
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-sc-cinderplugin
+provisioner: cinder.csi.openstack.org
+parameters:
+  type: multiattach
```

Comment:
I'm wondering why it's related to volume type? I thought it's related to `accessModes` when defining PersistentVolumeClaim.

Reply:
@lingxiankong right , there are 2 ways of doing it (either way creating volume type is required if we need to attach to more than one node) 
1. Support automatically create volume type of multiattach , depending on the access modes that are specified in pvc (ReadWriteMany, ReadOnlyMany..).
2. Use the existing field of volume type we already support and create volume of that type. 

So, in the initial support we provide in k8s, I thought it would be better to leave it for operator if they really would like to create the volume of type multiattach,(understanding their limitations ) instead of creating it automatically for them which might cause issues later on . 

https://docs.openstack.org/cinder/latest/admin/blockstorage-volume-multiattach.html#ro-rw-caveats-the-secondary-rw-attachment-issue .

Reply:
I think leave to operator to decide is a better option here..
if customer has requirement on option 1) above, we can consider it later on